renovate.json

```diff
@@ -46,5 +46,33 @@
       ],
       "automerge": true
     }
+  ],
+  "regexManagers": [
+    {
+      "fileMatch": ["^docs/tutorials/ofo.md$"],
+      "matchStrings": ["open-feature\\/open-feature-operator\\/releases\\/download\\/(?<currentValue>.*?)\\/release\\.yaml"],
+      "depNameTemplate": "open-feature/open-feature-operator",
+      "datasourceTemplate": "github-releases"
+    },
+    {
+      "fileMatch": ["^docs/tutorials/getting-started/java.mdx$"],
+      "matchStringsStrategy": "any",
+      "matchStrings": [
+        "\\s*<groupId>dev\\.openfeature<\\/groupId>\\s*<artifactId>sdk<\\/artifactId>\\s*<version>(?<currentValue>.*?)<\\/version>",
+        "'dev\\.openfeature:sdk:(?<currentValue>.*?)'"
+      ],
+      "depNameTemplate": "dev.openfeature:sdk",
+      "datasourceTemplate": "maven"
+    },
+    {
+      "fileMatch": ["^docs/tutorials/getting-started/java.mdx$"],
+      "matchStringsStrategy": "any",
+      "matchStrings": [
+        "\\s*<groupId>dev\\.openfeature\\.contrib\\.providers<\\/groupId>\\s*<artifactId>flagd<\\/artifactId>\\s*<version>(?<currentValue>.*?)<\\/version>",
+        "'dev\\.openfeature\\.contrib\\.providers:flagd:(?<currentValue>.*?)'"
+      ],
+      "depNameTemplate": "dev.openfeature.contrib.providers:flagd",
+      "datasourceTemplate": "maven"
+    }
```

Comment:
Is there a way to restrict this update so that it only applies to minor/patch versions? 

I'm worried that if we update major version, we may break our examples with breaking changes.

@beeme1mr @skyerus do you guys have any opinions on that? Am I worried about nothing?

Reply:
This won't auto-merge so I think it's okay.

Reply:
I think that as long as auto-merge is disabled this is fine. We may want to include a major upgrade if the breaking change doesn't affect the tutorial in question or perhaps even integrate it if it does. The PR being raised is a reminder to check this.